### DISCUSSION TITLE
Wireguard and CGNAT

### DISCUSSION THREAD

**Discussion by drtiwarir** [angristan/wireguard-install#469] [Help]
I am very thankful to Angristan for making 1 click wireguard server installation.
I used it to watch streaming videos/movies/sports from India, but recently Indian websites have blocked all data centre IP addresses.
I have a old pc with Ubuntu at my home and want to install wireguard server on it, but my home broadband is behind CGNAT.
I have searched on github and tried but no good results.
Angristan can you help with a 1 click script for VPS (data centre) and 1 click script for home PC (behind CGNAT) ?

Kind regards

**Comment by Enfernuz** [comment]
I don't think there is a 1-click script but at least there is a working set of configs made by @smbm to overcome CGNAT issues: https://github.com/smbm/wireguard-cgnat-bypass

It worked fine for me when I tried it.

One more resource with actual "1-click" scripts for setting up a supplementary WG node on Digital Ocean and Oracle Cloud (I have not tried it though): https://github.com/mochman/Bypass_CGNAT

**Comment by drtiwarir** [comment]
Thanks
I had already worked it with AWS

**Comment by UnderEu** [comment]
It works on the current Internet Protocol, you don't need to rely on ancient technology.
